lmfdb/characters/test_characters.py

```diff
@@ -169,5 +159,5 @@ def test_dirichletchar9999lfunc(self):
     def test_dirichletchar99999999999999999lfunc(self):
         """ Check Dirichlet character with very large modulus"""
         W = self.tc.get('/Character/Dirichlet/99999999999999999999/2')
-        assert parity_string(-1) in W.get_data(as_text=True) and '536870912' in W.get_data(as_text=True)
```

Comment:
Since we're removing the `Galois orbit` section in the large modulus case, this test failed, so as above I changed it to something that *is* there. Again do let me know if it should be otherwise.

Reply:
I agree with removing the Galois orbit section for characters of modulus > 10000 which are not in the database, since we don't actually know the orbit label.